### PR TITLE
Adding images in the repo itself

### LINKED ISSUE #2908: Move any image assets in the documentation to repository
The image in the Readme is loading from https://storage.googleapis.com/eventyay.com/assets/branding/frontend_branding.png

Please move it to the /docs/image/ folder and store all images in the documentation in the repo itself.

### PR DESCRIPTION
**Pull request by kushthedude**
Fixes #2908

### CODE REVIEW

**Comment by kushthedude** [comment]
@mariobehling @CosmicCoder96 Please Review

**Comment by kushthedude** [comment]
@mariobehling Have a look now please .

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Open Event Frontend
-![Open Event Frontend](https://storage.googleapis.com/eventyay.com/assets/branding/frontend_branding.png)
+![Open Event Frontend](public/images/Frontend_Landing.png)
```
Comment (pradeepgangwar) [inline README.md]: IMO moving the image to docs folder will be better since it belongs to documentation area.

**Comment by mariobehling** [comment]
Please read the issue. All documentation resources go into /docs.

**Comment by kushthedude** [comment]
Sorry , for mistake pushing commit asap

File: README.md (hunk 1 above)
Comment (kushthedude) [reply README.md]: Have a look now @pradeepgangwar

**Pull request merged**